config_defaults_inc.php

```diff
@@ -2697,7 +2697,17 @@
 
 /**
  * access level needed to be able to be listed in the assign to field.
- * @global integer $g_handle_bug_threshold
+ * if a single value provided, this means that all users that have 
+ * access level >= this value, will be listed.
+ * example:
+ * $g_handle_bug_threshold = DEVELOPER;
+ * 
+ * if an array is provided, then user access level need to be in the set
+ * provided in the array, for the user to be listed.
```

Comment:
* Start sentence with capital letter.
* _then **the** user **'s** access level need**s**_
* provided -> defined